src/main/clojure/clara/tools/testing_utils.clj

```diff
@@ -75,3 +75,9 @@
   (f)
   (with-redefs [uc/get-ordered-update-cache ca/get-cancelling-update-cache]
     (f)))
+
```

Comment:
Would you recommend turning it into a cljc file or moving that function to a different namespace?